torch/_dynamo/variables/tensor.py

```diff
@@ -973,6 +970,9 @@ def call_method(
 
         return result
 
+    def python_type(self):
+        return np.ndarray
+
```

Comment:
This seems not necessary

Reply:
This was necessary to make the test in that commit pass. It used to pass just by chance, because, for some reason, the array was being parsed rather than as a `NumpyNdarrayVariable`, simply as a `NumpyVariable`, and the `isinstance` was going through a path that was returning `True` pretty much by chance.

Reply:
Also, note that, the way this was implemented in `TensorVariable` it was returning the python type of a `NumpyNdarrayVariable` to be `torch.Tensor`, which was not correct in any way.